crates/ink/codegen/Cargo.toml

```diff
@@ -30,6 +30,7 @@ blake2 = { workspace = true }
 heck = { workspace = true }
 scale = { workspace = true }
 impl-serde = { workspace = true, default-features = true }
+hex = { workspace = true }
```

Comment:
We already use `impl_serde` (renamed to serde_hex`) for hex encoding/decoding, no need to have two different libs for this.